Fatima-Paola.js

```diff
@@ -0,0 +1,122 @@
+//question 1, Calculate the Cube
+
+const calculateCube = (num) => {
+    console.log(num*num*num);
+  };
+  calculateCube(5);
+  
+  
+  //question 2, Is a Vowel?
+  
+  const letters = ['a','e','i','o','u'];
+  
+   const isAVowel = (vowels) => {
+    if (letters.includes(vowels)) {
+      console.log("true");
+    } else {
+      console.log("false");
+    }
+   }
+  console.log(isAVowel("a"));
+  
+  // const isAVowel = (letter) => {
+  //   if (letter === "a" ||"e" || "i" ||"o" || "u") {
+  //     console.log("true");
+  //   } else {
+  //     console.log("false")
+  //   }
+  // console.log(isAVowel("a");
+  //this this might work but nooooopppeeee
+  
+  
+  //question 3, get two lengths
+  
+  const getTwoLengths = (parameter1, parameter2) =>
+  [parameter1.length, parameter2.length]; 
```

Comment:
this works, but for readability it's important to put the curly brackets after the arrow